lib/core/src/sdk.rs

```diff
@@ -1969,39 +2007,105 @@ mod tests {
                     ChainSwapStates::TransactionMempool,
                     ChainSwapStates::TransactionConfirmed,
                 ] {
-                    let mock_tx = create_mock_tx(sdk.onchain_wallet.clone()).await.unwrap();
-                    let mock_lockup_tx_id = mock_tx.txid().to_string();
-
+                    let mock_tx = TEST_LIQUID_TX.clone();
+                    let mock_tx_id = mock_tx.txid().to_string();
```

Comment:
Should we test this the same way (but opposite) as testing TransactionServerMempool/TransactionServerConfirmed? So:
- Direction::Incoming => TEST_BITCOIN_TX
- Direction::Outgoing => TEST_LIQUID_TX